components/automate-ui/src/app/modules/team/team-details/team-details.component.ts

```diff
@@ -228,15 +205,14 @@ export class TeamDetailsComponent implements OnInit, OnDestroy {
 
   onProjectDropdownClosing(selectedProjects: string[]): void {
 
-    Object.keys(this.projects).forEach(id => this.projects[id].checked = false);
-    selectedProjects.forEach(id => this.projects[id].checked = true);
+    this.updateForm.controls.projects.setValue(selectedProjects);
 
     // since the app-projects-dropdown is not a true form input (select)
     // we have to manage the form reactions
-    if (noProjectsUpdated(this.team.projects, this.projects)) {
-      this.updateNameForm.controls.projects.markAsPristine();
+    if (xor(this.team.projects, this.updateForm.controls.projects.value).length === 0) {
+      this.updateForm.controls.projects.markAsPristine();
     } else {
-      this.updateNameForm.controls.projects.markAsDirty();
+      this.updateForm.controls.projects.markAsDirty();
```

Comment:
The form field was not even being used previously--except to track dirty/pristine!